docs/platforms/apple/common/session-replay/index.mdx

```diff
@@ -20,6 +20,30 @@ Before enabling Session Replay in production, verify your masking configuration
 
 </Alert>
 
+<Alert level="warning">
+
+<strong>Session Replay is disabled by default on iOS 26.0+ with Xcode 26.0+ to prevent PII leaks</strong>
+
+Due to potential masking issues introduced by Apple's Liquid Glass rendering changes in iOS 26.0, Session Replay is now <strong>automatically disabled</strong> on apps running iOS 26.0+ when built with Xcode 26.0 or later starting with Sentry Cocoa <strong>8.57.0</strong>. This is a defensive measure to protect user privacy and prevent potential PII leaks until masking is reliably supported. Earlier SDK versions do not include this safeguard and may be affected by unreliable masking on iOS 26.0 when built with Xcode 26.0 (Liquid Glass). We recommend upgrading to 8.57.0+.
```

Comment:
`m`: I think we could also point out what users should do when they can't upgrade the Cocoa SDK right now.
```suggestion
Due to potential masking issues introduced by Apple's Liquid Glass rendering changes in iOS 26.0, Session Replay is now <strong>automatically disabled</strong> on apps running iOS 26.0+ when built with Xcode 26.0 or later starting with Sentry Cocoa <strong>8.57.0</strong>. This is a defensive measure to protect user privacy and prevent potential PII leaks until masking is reliably supported. Earlier SDK versions do not include this safeguard and may be affected by unreliable masking on iOS 26.0 when built with Xcode 26.0 (Liquid Glass). We recommend upgrading to 8.57.0+ or, if you can't upgrade the SDK, disabling Session Replay manually.
```